src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_skeram.cpp

```diff
@@ -84,6 +85,12 @@ class boss_skeram : public CreatureScript
                     ((TempSummon*)me)->UnSummon();
             }
 
+            void SpellHitTarget(Unit* target, SpellInfo const* spell) override
+            {
+                if (spell->Id == SPELL_TRUE_FULFILLMENT)
+                    target->CastSpell(target, SPELL_TRUE_FULFILLMENT_2, true, NULL, NULL, me->GetGUID());
```

Comment:
nullptr - optional change - you can do without it.
caster should not equal target - why? What's the difference?

@Unknown-1000 - you can close and solve problem with sql part.
``` SQL
DELETE FROM `spell_linked_spell` WHERE `spell_trigger` = 784;
INSERT INTO `spell_linked_spell` (`spell_trigger`, `spell_effect`, `type`, `comment`) VALUES
(784,2313,1,'True Fulfillment Charm - Cast True Fulfillment Resistance part.');
```